ui/package-lock.json

```diff
@@ -16,7 +16,7 @@
                 "@fortawesome/react-fontawesome": "^0.1.11",
                 "@testing-library/jest-dom": "^5.16.4",
                 "@testing-library/react": "^12.1.5",
-                "bootstrap": "^5.3.3",
+                "bootstrap": "^5.0.0",
```

Comment:
please revert this

Reply:
Not only will I be reverting this, but I will do add a list to the description of any file that isn't *.java.

Thanks for catching

Reply:
Done